docs/ase-file-specs.md

```diff
@@ -251,13 +262,13 @@ Color profile for RGB or grayscale values.
 
 ### External Files Chunk (0x2008)
 
-A list of external files linked with this file. It might be used to
-reference external palettes or tilesets.
+A list of external files linked with this file can be found in the first frame. It might be used to
+reference external palettes, tilesets, or extensions that make use of extended properties.
 
     DWORD       Number of entries
     BYTE[8]     Reserved (set to zero)
     + For each entry
-      DWORD     Entry ID (this ID is referenced by tilesets or palettes)
+      DWORD     Entry ID (this ID is referenced by tilesets, palettes, or extended properties)
       BYTE[8]   Reserved (set to zero)
```

Comment:
Probably we'll need to use one of these reserved bytes to specify the kind of external file (the idea of this chunk was to have a way to pre-load external files, but in this case for extensions it's for other usage, so we'll need a way to make a difference)

Reply:
I was appending an "extension:" prefix to differentiate them. But sure, I can use one of those reserved bytes if you think it would be better. Just let me know if you prefer to use a reserved byte.

Reply:
We can use one BYTE for the type (0 external palette, 1 external tileset, 2 extension name for properties)

Reply:
👍